receiver/simpleprometheusreceiver/testdata/config.yaml

```diff
@@ -4,6 +4,8 @@ receivers:
     endpoint: "localhost:1234"
     collection_interval: 30s
     metrics_path: /v2/metrics
+    params:
+      columns: "name,messages"
```

Comment:
```suggestion
      columns: [name, messages]
```

Reply:
OK, that's not a good example config - I'll change it, but your suggestion breaks tests. I'll have to rework this a bit.

Reply:
Added one more parameter - both comma-delimited and arrays are supported.